src/index.ts

```diff
@@ -244,6 +254,61 @@ const tracker: JupyterFrontEndPlugin<void> = {
   }
 };
 
+/*
+ * A plugin for add widget with detial of veriable.
+ */
+const details: JupyterFrontEndPlugin<void> = {
+  id: '@jupyterlab/debugger:details',
+  autoStart: true,
+  requires: [IDebugger],
+  activate: (app: JupyterFrontEnd, service: IDebugger) => {
+    const { commands, shell } = app;
+    const tracker = new WidgetTracker<MainAreaWidget<VariableDetails>>({
+      namespace: 'variableDetails'
+    });
+
+    commands.addCommand(CommandIDs.addDetails, {
+      label: 'Continue',
+      caption: 'Continue',
+      execute: async args => {
+        const { variableReference } = args;
+        const title = args.title as string;
+        if (!variableReference || variableReference === 0) {
+          return;
+        }
+        const details = await service.getVariableDetails(
+          variableReference as number
+        );
+
+        if (
+          !details ||
+          details.length === 0 ||
+          tracker.find(widget => widget.id === `jp-debugger-details-${title}`)
+        ) {
+          return;
+        }
+
+        const model = (service.model as DebuggerModel).variables;
+        const openOption: DocumentRegistry.IOpenOptions = {
+          mode: tracker.currentWidget ? 'split-right' : 'split-bottom'
+        };
+        const widget = new MainAreaWidget<VariableDetails>({
+          content: new VariableDetails({
+            commands,
+            service,
+            details,
+            model,
+            title
+          })
+        });
+        widget.id = `jp-debugger-details-${title}`;
```

Comment:
This refers to a CSS class that does not exist anymore?

Reply:
No, that's id needed for adding to shell and for tracking. Do you think we should use a different method of creating id for these widgets?

Reply:
Ah ok I thought there was a link to the previous CSS class (see https://github.com/jupyterlab/debugger/pull/347#discussion_r373555799).

Reply:
Yeah, that id is similar for last css style.